fem/bilininteg_vectorfe.cpp

```diff
@@ -511,8 +536,10 @@ void VectorFEMassIntegrator::AssemblePA(const FiniteElementSpace &trial_fes,
 
    symmetric = MQ ? MQ->IsSymmetric() : true;
 
-   if (trial_fetype == mfem::FiniteElement::CURL &&
-       test_fetype == mfem::FiniteElement::DIV && dim == 3)
+   if (dim == 3 && ((trial_fetype == mfem::FiniteElement::CURL &&
+                     test_fetype == mfem::FiniteElement::DIV) ||
+                    (trial_fetype == mfem::FiniteElement::DIV &&
+                     test_fetype == mfem::FiniteElement::CURL)))
```

Comment:
```suggestion
   const bool trial_curl = (trial_fetype == mfem::FiniteElement::CURL);
   const bool trial_div = (trial_fetype == mfem::FiniteElement::DIV);
   const bool test_curl = (test_fetype == mfem::FiniteElement::CURL);
   const bool test_div = (test_fetype == mfem::FiniteElement::DIV);
   if (dim == 3 && ((trial_curl && test_div) || (trial_div && test_curl)))
```

Reply:
I think defining these additional booleans would simplify reading the remaining `if` / `else` statements. There are several additional places where these booleans could be used here that weren't modified in the commit (so I can't suggest changes in those lines).

Reply:
Thanks @stefanhenneking for the suggestions, which are now committed.